README.md

```diff
@@ -1,9 +1,85 @@
-INSTRUCTIONS FOR SETTING UP:
+# WHAT IS THIS?
 
-To access the environment variables you need to:
+This is a project that makes use of PostgreSQL and the express framework to create a database and an API to retrieve information from that database.
 
-1. create a file with the name: .env.test
-2. inside .env.test you need to write: PGDATABASE=nc_snacks_test
+PostgreSQL is the database management system used in this project to create the database and store the data
 
-3. create a file with the name: .env.development
-4. inside .env.development you need to write: PGDATABASE=nc_snacks
+Express is the framework that has been used to create the server to which clients send requests to in order to retrieve information from the PostgreSQL database
+
+This project makes use of node postgres which is what allows the server to query the database and retrieve the relevant data to send back to the client
+
+This project has been thoroughly tested using jest and supertest in order to ensure that the API works correctly and efficiently and most importantly is free from vulnerabilities
+
+--------------------------------------------------------------------------------------------------------------------------------------------
+
+
+# INSTRUCTIONS FOR INTERACTING WITH THE API
+
+- To view the hosted version of this api, head to the following link: https://nc-games-ki7a.onrender.com/
```

Comment:
Just a comment - I would probably make this bold or stand out in some way as just gazing over this I struggled to find the link. Maybe put the link higher up on the page? Just an idea to make it more obvious to potential employers looking at your read me :)